python.d/unbound.chart.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# Description: unbound netdata python.d module
+# Author: Austin S. Hemmelgarn (Ferroin)
+
+import os
+import yaml
```

Comment:
Netdata shipped with `yaml` packages for py2, py3.
If the only thing you need is `yaml.load` i suggest to use 
https://github.com/l2isbad/netdata/blob/master/python.d/python_modules/bases/loaders.py#L32-L50

Reply:
Oh, completely missed that.  I'll get it updated to use that then.

Reply:
On second inspection, it looks like that won't be quite enough, as I kind of need to sanely catch parsing errors (especially since they're reasonably likely due to tab indentation).

Would a construct like what's used in the file you linked to import the requisite libraries work here?

Reply:
Should the `check()` return true if an `yaml` parse error occurs? (If config exist and readable but has wrong `yaml` syntax)

Reply:
Yes, in general it probably should.  The actual check being made is that a connection works.  If we don't have any permission to access the unix socket, or we don't have a valid client certificate configured, the connect will fail, and in turn the check will fail.

The attempt to parse the unbound config is simply there to try and auto-fill config, but anything specified manually (except the port currently, which gets overridden either way if we try to parse the unbound config, though that probably needs to change) will override that.

If for example you manually specify the certificate and key, but specify a bogus unbound config (or the config parsing fails because of the tab s most distros have in their default unbound config), we should just fall back to trying on port 8953 on the loopback interface.